set.mm

```diff
@@ -28830,29 +28860,32 @@ choice between (what we call) a "definitional form" where the shorter
     r19.29af.0 $e |- F/ x ph $.
     r19.29af.1 $e |- ( ( ( ph /\ x e. A ) /\ ps ) -> ch ) $.
     r19.29af.2 $e |- ( ph -> E. x e. A ps ) $.
-    $( A commonly used pattern based on ~ r19.29 .  (Contributed by Thierry
+    $( A commonly used pattern based on ~ r19.29 .  See ~ r19.29a , ~ r19.29an
+       for a variant when ` x ` is disjoint from ` ph ` (Contributed by Thierry
        Arnoux, 29-Nov-2017.) $)
```

Comment:
```suggestion
       for a variant when ` x ` is disjoint from ` ph ` .  (Contributed by
       Thierry Arnoux, 29-Nov-2017.) $)
```

(also, maybe move ~ ralbiim along with ~ ralbi ?)

otherwise approve

Reply:
> move ~ ralbiim along with ralbi

ralbiim needs r19.26, so, unfortunately, this is not easily possible.

Thanks for the hint to the typo.  Fixed.